tests/test_transformers.py

```diff
@@ -386,3 +387,60 @@ def test_DFStandardScaler_works_in_pipeline_with_DFFeatureUnion(categorical, num
     result = pipeline.fit_transform(numerical)
 
     pd.testing.assert_frame_equal(result, numerical_scaled)
+
+
+def test_dfrowfunc_no_strategy(numerical_na):
+    with pytest.raises(TransformerError,
+                       message="Expecting TransformerError but no error occurred",
+                       match="No strategy is specified."):
+        DFRowFunc()
+
+
+def test_dfrowfunc_strategy_is_string_not_in_map(numerical_na):
+    strategy = 'avg'
+
+    with pytest.raises(TransformerError,
+                       message="Expecting TransformerError but no error occurred",
+                       match=f"Strategy {strategy} is not a the predefined strategy."):
+        DFRowFunc(strategy=strategy)
```

Comment:
Would just call DFRowFunc directly. Then it should appear directly in the traceback if the test fails

```suggestion
        DFRowFunc(strategy='avg')
```

Reply:
Alternatively, combine these tests using pytest.mark.parametrize
```python
@pytest.mark.parametrize('strategy, message', [
               ('avg', "Strategy avg is not a predefined strategy"), 
               (1337, "1337 is not a callable or a string")
])
def test_dfrowfunc_strategy_raises(numerical_na, strategy, message):
    with pytest.raises(TransformerError, match=message):
            raise DFRowFunc(strategy=strategy)
```